src/services/railgun/railgun-txids/__tests__/railgun-txid-sync-graph-v2.test.ts

```diff
@@ -18,6 +18,11 @@ describe('railgun-txid-sync-graph', () => {
       launchBlock: 1000,
       launchTimestamp: 1000,
     };
+
+    NETWORK_CONFIG[NetworkName.EthereumSepolia].poi = {
```

Comment:
I think this should match shared-models: https://github.com/Railgun-Community/shared-models/blob/3baba0e95f2dea5ff3e61f82bbdadc4bd8cc49b9/src/models/network-config.ts#L502